public/locales/en/common.json

```diff
@@ -82,9 +83,9 @@
     "MAIN_COURSE": "Main Course"
   },
   "FACILITATORS": {
-    "ALL_STATES": "States",
-    "ALL_DISTRICTS": "Districts",
-    "ALL_BLOCKS": "Blocks",
+    "ALL_STATES": "State",
```

Comment:
@upendraTekdi Key should be changed accordingly

Reply:
done to 
 "STATE": "State",
    "DISTRICT": "District",
    "BLOCK": "Block",

Reply:
@upendraTekdi cant see here